typings/globals.d.ts

```diff
@@ -0,0 +1,29 @@
+type Struct = NodeJS.Dict<any>;
+type Dataset = Array<Struct>;
```

Comment:
I'd prefer to have each `Struct`, `Dataset` etc be a generic with the ability to specify a type as in its current form it is no different from having no typings since it will always be `any` anyway.

Reply:
We will use typings just for IDE autocomplete and hints, not for compiling. We can't predict certain collection structure so we can't define it more precise.